src/charm.lua

```diff
@@ -1191,13 +1229,78 @@ if next(SMODS.find_mod 'Cryptid') then
     local raw_get_weight = SMODS.Rarities.Common.get_weight
     SMODS.Rarity:take_ownership("Common", {
         get_weight = function(...)
-            return G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_DuctTape' and 0 or raw_get_weight(...)
+            return G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_DuctTape'
+                and not is_charm_debuffed_safe() and 0 or raw_get_weight(...)
         end
     })
     local raw_get_weight = SMODS.Rarities.Common.get_weight
     SMODS.Rarity:take_ownership("Uncommon", {
         get_weight = function(...)
-            return G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_DuctTape' and 0 or raw_get_weight(...)
+            return G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_DuctTape'
+                and not is_charm_debuffed_safe() and 0 or raw_get_weight(...)
+        end
+    })
+end
+
+if next(SMODS.find_mod 'GARBPACK') then -- Garbshit
+    Bakery_API.credit(Bakery_API.Charm {
+        key = 'Virus',
+        pos = {
+            x = 4,
+            y = 2
+        },
+        atlas = 'Charms',
+        artist = 'Jack5',
+        coder = 'Jack5',
+        unlocked = false,
+        -- TODO: Fix "ERROR" displaying in right pane when unlocking
```

Comment:
This doesn't happen for me, so it's probably a mod compat issue (which should still be fixed).